<html>
<head><meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>MetaGO result for PF13617</title></head>
<body>
<script type="text/javascript" src="/jmol/JSmol.min.js"></script>
<script type="text/javascript" src="/jmol/Jmol2.js"></script>
[<a href="http://zhanglab.ccmb.med.umich.edu/MetaGO">back to server</a>]<br/>
<h1 align="center">MetaGO result for job id PF13617</h1>


<font size="4" face="Arial">

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;User Input</font></b></div>
<ul>
    <table style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td colspan="2">&gt;UniRef100_A0A085AIJ0/8-63 (56 residues)<br>
    LCALAAMMLTGCTPRIEVAAPKEPITINMNVKIEHEIHIKVDKDVENLLKSRSDLF<br>
    </td></tr>
    </table><br>

    <font size=2><br>
    Download query <a href="seq.fasta" download>sequence</a> 
    and <a href="input.pdb.gz" download>structure</a><br>
    </font>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Proteins with similar structure</font></b></div>
<ul>
    <table border="0" style="background:#F2F2F2";align="left">
        <tr><td align ="center"><script type="text/javascript">
            jmolInitialize("/jmol");
            jmolSetAppletColor("#000000");
            jmolApplet(450, "load model1_1na6B1.pdb.gz;set frank off;select all;spacefill off;wireframe off;select :T;trace 0.35;color trace mediumpurple;select :Q;cartoons;color group;set ambient 40;");
        </script><br/><script type="text/javascript">
            jmolCheckbox("set spin x 10;set spin y 10;spin on","spin off","Spin On/Off",false)
        </script></td>
        <td valign="top"><b>Top 10 structural analogs in PDB (as identified by
        <a href="http://zhanglab.ccmb.med.umich.edu/TM-align" target="_blank">
        TM-align</a>)</b><br/><br/>
        <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><th>Click<br>to view</th><th>Rank</th><th>PDB<br/>Hit</th><th>TM-score</th><th>RMSD<sup>a</sup></th><th>IDEN<sup>a</sup></th><th>Cov.</th><th>Download<br/>Alignment</th></tr>
        
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_1na6B1.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">1</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1na6" target="_blank"><font face="Monospace">1na6B1</font></a></td><td align="center">0.69</td><td valign="middle" align="center">2.32</td><td valign="middle" align="center">0.071</td><td valign="middle">1.000</td><td><a href="model1_1na6B1.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_5c0wK.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">2</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5c0w" target="_blank"><font face="Monospace">5c0wK</font></a></td><td align="center">0.65</td><td valign="middle" align="center">1.71</td><td valign="middle" align="center">0.191</td><td valign="middle">0.839</td><td><a href="model1_5c0wK.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_3safA1.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">3</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3saf" target="_blank"><font face="Monospace">3safA1</font></a></td><td align="center">0.64</td><td valign="middle" align="center">1.78</td><td valign="middle" align="center">0.085</td><td valign="middle">0.839</td><td><a href="model1_3safA1.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_2hbkA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">4</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2hbk" target="_blank"><font face="Monospace">2hbkA</font></a></td><td align="center">0.64</td><td valign="middle" align="center">1.84</td><td valign="middle" align="center">0.188</td><td valign="middle">0.857</td><td><a href="model1_2hbkA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_4nlbA1.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">5</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4nlb" target="_blank"><font face="Monospace">4nlbA1</font></a></td><td align="center">0.63</td><td valign="middle" align="center">1.90</td><td valign="middle" align="center">0.146</td><td valign="middle">0.857</td><td><a href="model1_4nlbA1.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_3cymA1.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">6</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3cym" target="_blank"><font face="Monospace">3cymA1</font></a></td><td align="center">0.61</td><td valign="middle" align="center">2.56</td><td valign="middle" align="center">0.058</td><td valign="middle">0.911</td><td><a href="model1_3cymA1.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_1yt3A1.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">7</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1yt3" target="_blank"><font face="Monospace">1yt3A1</font></a></td><td align="center">0.59</td><td valign="middle" align="center">2.10</td><td valign="middle" align="center">0.128</td><td valign="middle">0.839</td><td><a href="model1_1yt3A1.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_4jvwA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">8</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4jvw" target="_blank"><font face="Monospace">4jvwA</font></a></td><td align="center">0.47</td><td valign="middle" align="center">2.58</td><td valign="middle" align="center">0.227</td><td valign="middle">0.714</td><td><a href="model1_4jvwA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_2wl3A.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">9</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2wl3" target="_blank"><font face="Monospace">2wl3A</font></a></td><td align="center">0.46</td><td valign="middle" align="center">2.26</td><td valign="middle" align="center">0.132</td><td valign="middle">0.696</td><td><a href="model1_2wl3A.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_2qfzA2.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">10</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2qfz" target="_blank"><font face="Monospace">2qfzA2</font></a></td><td align="center">0.42</td><td valign="middle" align="center">3.56</td><td valign="middle" align="center">0.071</td><td valign="middle">0.768</td><td><a href="model1_2qfzA2.pdb.gz" download>Download</a></td></tr>
        </table><br/>

        <table border="0" cellspacing="2" style="font-family:Arial;font-size:12px;background:#F2F2F2;">
            <tr><td valign="top">(a)</td><td align="justify">Query structure is shown in cartoon, while the structural analog is displayed using backbone trace.</td></tr>
            <tr><td valign="top">(b)</td><td align="justify">Ranking of proteins is based on TM-score of the structural alignment between the query structure and known structures in the PDB library.</td></tr>
            <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
            <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
            <tr><td valign="top">(e)</td><td align="justify">Cov. represents the coverage of the alignment by TM-align and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
        </table>
    </td></tr>
    </table>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Gene Ontology (GO) Terms</font></b></div>
<ul>
<table border="0" style="background:#F2F2F2";align="left"><tr><td>

<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_MF.svg" target="_blank">
    <img width=450 height=450 border="1" style="border-color:black" alt="GOsearchresult_final_MF.svg" src="GOsearchresult_final_MF.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Molecular Function (MF)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:1901265" target="_blank">GO:1901265</a></td><td align="center">0.02</td><td>nucleoside phosphate binding</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0036094" target="_blank">GO:0036094</a></td><td align="center">0.02</td><td>small molecule binding</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0016888" target="_blank">GO:0016888</a></td><td align="center">0.02</td><td>endodeoxyribonuclease activity, producing 5'-phosphomonoesters</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0015666" target="_blank">GO:0015666</a></td><td align="center">0.02</td><td>restriction endodeoxyribonuclease activity</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0016896" target="_blank">GO:0016896</a></td><td align="center">0.01</td><td>exoribonuclease activity, producing 5'-phosphomonoesters</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_MF.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Molecular Function. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.13,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>


<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_BP.svg" target="_blank">
    <img width=450 height=450 border="1" style="border-color:black" alt="GOsearchresult_final_BP.svg" src="GOsearchresult_final_BP.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Biological Process (BP)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0071047" target="_blank">GO:0071047</a></td><td align="center">0.02</td><td>polyadenylation-dependent mRNA catabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0071046" target="_blank">GO:0071046</a></td><td align="center">0.02</td><td>nuclear polyadenylation-dependent ncRNA catabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0071041" target="_blank">GO:0071041</a></td><td align="center">0.02</td><td>antisense RNA transcript catabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0071034" target="_blank">GO:0071034</a></td><td align="center">0.02</td><td>CUT catabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0071033" target="_blank">GO:0071033</a></td><td align="center">0.02</td><td>nuclear retention of pre-mRNA at the site of transcription</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0071031" target="_blank">GO:0071031</a></td><td align="center">0.02</td><td>nuclear mRNA surveillance of mRNA 3'-end processing</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044355" target="_blank">GO:0044355</a></td><td align="center">0.02</td><td>clearance of foreign intracellular DNA</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0034472" target="_blank">GO:0034472</a></td><td align="center">0.02</td><td>snRNA 3'-end processing</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0031126" target="_blank">GO:0031126</a></td><td align="center">0.02</td><td>snoRNA 3'-end processing</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0031125" target="_blank">GO:0031125</a></td><td align="center">0.02</td><td>rRNA 3'-end processing</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_BP.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Biological Process. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.08,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>


<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_CC.svg" target="_blank">
    <img width=450 height=450 border="1" style="border-color:black" alt="GOsearchresult_final_CC.svg" src="GOsearchresult_final_CC.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Cellular Component (CC)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0035327" target="_blank">GO:0035327</a></td><td align="center">0.01</td><td>transcriptionally active chromatin</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0005622" target="_blank">GO:0005622</a></td><td align="center">0.01</td><td>intracellular</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0000176" target="_blank">GO:0000176</a></td><td align="center">0.01</td><td>nuclear exosome (RNase complex)</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_CC.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Cellular Component. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.14,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>

</td></tr>
</table>
</ul>




</font>


<hr>
<b>Reference:</b>
<ul><li>Chengxin Zhang, Peter L Freddolino, and Yang Zhang. (2018) <i>MetaGO: A Structure and Protein-Protein Interaction Network Based Pipeline for Automated Protein Function Annotations</i>. Journal of Molecular Biology.</ul>
[<a href="http://zhanglab.ccmb.med.umich.edu/MetaGO">back to server</a>]<br/>
</body></html>
