<html>
<head><meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>MetaGO result for PF13865</title></head>
<body>
<script type="text/javascript" src="/jmol/JSmol.min.js"></script>
<script type="text/javascript" src="/jmol/Jmol2.js"></script>
[<a href="http://zhanglab.ccmb.med.umich.edu/MetaGO">back to server</a>]<br/>
<h1 align="center">MetaGO result for job id PF13865</h1>


<font size="4" face="Arial">

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;User Input</font></b></div>
<ul>
    <table style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td colspan="2">&gt;UniRef100_UPI000333F8DE/1-248 (248 residues)<br>
    MAAQSAPKVVLKSTTKMSLNERFTNMLKNKQPMPVNIRASMQQQQQLASARNRRLAQQME<br>
NRPSVQAALKLKQSLKQRLGKSNIQARLGRPIGALARGAVGGRGLPIIQRGLPRGALRGG<br>
RATRTLLRGGMSLRGQNLLRGGRAAAPRMGLRRGGVRGRGGPGRGGLGRGAMGRGGISGR<br>
GRGIIGRGRGGFGGRGRGRGRGRGALSRPVLTKEQLDNQLDAYMSKTKGHLDAELDAYMA<br>
QTDPEAND<br>
    </td></tr>
    </table><br>

    <font size=2><br>
    Download query <a href="seq.fasta" download>sequence</a> 
    and <a href="input.pdb.gz" download>structure</a><br>
    </font>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Proteins with similar structure</font></b></div>
<ul>
    <table border="0" style="background:#F2F2F2";align="left">
        <tr><td align ="center"><script type="text/javascript">
            jmolInitialize("/jmol");
            jmolSetAppletColor("#000000");
            jmolApplet(450, "load model1_3nvnB.pdb.gz;set frank off;select all;spacefill off;wireframe off;select :T;trace 0.35;color trace mediumpurple;select :Q;cartoons;color group;set ambient 40;");
        </script><br/><script type="text/javascript">
            jmolCheckbox("set spin x 10;set spin y 10;spin on","spin off","Spin On/Off",false)
        </script></td>
        <td valign="top"><b>Top 10 structural analogs in PDB (as identified by
        <a href="http://zhanglab.ccmb.med.umich.edu/TM-align" target="_blank">
        TM-align</a>)</b><br/><br/>
        <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><th>Click<br>to view</th><th>Rank</th><th>PDB<br/>Hit</th><th>TM-score</th><th>RMSD<sup>a</sup></th><th>IDEN<sup>a</sup></th><th>Cov.</th><th>Download<br/>Alignment</th></tr>
        
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_3nvnB.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">1</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3nvn" target="_blank"><font face="Monospace">3nvnB</font></a></td><td align="center">0.45</td><td valign="middle" align="center">5.94</td><td valign="middle" align="center">0.072</td><td valign="middle">0.766</td><td><a href="model1_3nvnB.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_1b25A.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">2</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1b25" target="_blank"><font face="Monospace">1b25A</font></a></td><td align="center">0.45</td><td valign="middle" align="center">5.96</td><td valign="middle" align="center">0.052</td><td valign="middle">0.750</td><td><a href="model1_1b25A.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_3ayfA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">3</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3ayf" target="_blank"><font face="Monospace">3ayfA</font></a></td><td align="center">0.44</td><td valign="middle" align="center">5.68</td><td valign="middle" align="center">0.058</td><td valign="middle">0.726</td><td><a href="model1_3ayfA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_2y8nA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">4</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2y8n" target="_blank"><font face="Monospace">2y8nA</font></a></td><td align="center">0.44</td><td valign="middle" align="center">5.78</td><td valign="middle" align="center">0.051</td><td valign="middle">0.734</td><td><a href="model1_2y8nA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_6damA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">5</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=6dam" target="_blank"><font face="Monospace">6damA</font></a></td><td align="center">0.43</td><td valign="middle" align="center">6.17</td><td valign="middle" align="center">0.037</td><td valign="middle">0.766</td><td><a href="model1_6damA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_1jqnA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">6</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1jqn" target="_blank"><font face="Monospace">1jqnA</font></a></td><td align="center">0.43</td><td valign="middle" align="center">6.25</td><td valign="middle" align="center">0.057</td><td valign="middle">0.774</td><td><a href="model1_1jqnA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_3pflA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">7</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3pfl" target="_blank"><font face="Monospace">3pflA</font></a></td><td align="center">0.43</td><td valign="middle" align="center">5.91</td><td valign="middle" align="center">0.065</td><td valign="middle">0.722</td><td><a href="model1_3pflA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_6h02A.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">8</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=6h02" target="_blank"><font face="Monospace">6h02A</font></a></td><td align="center">0.43</td><td valign="middle" align="center">6.12</td><td valign="middle" align="center">0.044</td><td valign="middle">0.758</td><td><a href="model1_6h02A.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_5i2aA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">9</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5i2a" target="_blank"><font face="Monospace">5i2aA</font></a></td><td align="center">0.43</td><td valign="middle" align="center">5.81</td><td valign="middle" align="center">0.033</td><td valign="middle">0.714</td><td><a href="model1_5i2aA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_2nlzD.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">10</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2nlz" target="_blank"><font face="Monospace">2nlzD</font></a></td><td align="center">0.42</td><td valign="middle" align="center">5.63</td><td valign="middle" align="center">0.043</td><td valign="middle">0.710</td><td><a href="model1_2nlzD.pdb.gz" download>Download</a></td></tr>
        </table><br/>

        <table border="0" cellspacing="2" style="font-family:Arial;font-size:12px;background:#F2F2F2;">
            <tr><td valign="top">(a)</td><td align="justify">Query structure is shown in cartoon, while the structural analog is displayed using backbone trace.</td></tr>
            <tr><td valign="top">(b)</td><td align="justify">Ranking of proteins is based on TM-score of the structural alignment between the query structure and known structures in the PDB library.</td></tr>
            <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
            <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
            <tr><td valign="top">(e)</td><td align="justify">Cov. represents the coverage of the alignment by TM-align and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
        </table>
    </td></tr>
    </table>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Gene Ontology (GO) Terms</font></b></div>
<ul>
<table border="0" style="background:#F2F2F2";align="left"><tr><td>

<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_MF.svg" target="_blank">
    <img width=450 height=450 border="1" style="border-color:black" alt="GOsearchresult_final_MF.svg" src="GOsearchresult_final_MF.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Molecular Function (MF)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0003676" target="_blank">GO:0003676</a></td><td align="center">0.98</td><td>nucleic acid binding</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0003723" target="_blank">GO:0003723</a></td><td align="center">0.44</td><td>RNA binding</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044822" target="_blank">GO:0044822</a></td><td align="center">0.36</td><td>poly(A) RNA binding</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0003677" target="_blank">GO:0003677</a></td><td align="center">0.35</td><td>DNA binding</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043565" target="_blank">GO:0043565</a></td><td align="center">0.34</td><td>sequence-specific DNA binding</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0008327" target="_blank">GO:0008327</a></td><td align="center">0.31</td><td>methyl-CpG binding</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0003824" target="_blank">GO:0003824</a></td><td align="center">0.05</td><td>catalytic activity</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0016274" target="_blank">GO:0016274</a></td><td align="center">0.04</td><td>protein-arginine N-methyltransferase activity</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0035242" target="_blank">GO:0035242</a></td><td align="center">0.03</td><td>protein-arginine omega-N asymmetric methyltransferase activity</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0008469" target="_blank">GO:0008469</a></td><td align="center">0.03</td><td>histone-arginine N-methyltransferase activity</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_MF.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Molecular Function. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.13,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>


<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_BP.svg" target="_blank">
    <img width=450 height=450 border="1" style="border-color:black" alt="GOsearchresult_final_BP.svg" src="GOsearchresult_final_BP.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Biological Process (BP)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0065007" target="_blank">GO:0065007</a></td><td align="center">0.89</td><td>biological regulation</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0050789" target="_blank">GO:0050789</a></td><td align="center">0.89</td><td>regulation of biological process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0050794" target="_blank">GO:0050794</a></td><td align="center">0.84</td><td>regulation of cellular process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0019222" target="_blank">GO:0019222</a></td><td align="center">0.84</td><td>regulation of metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0060255" target="_blank">GO:0060255</a></td><td align="center">0.83</td><td>regulation of macromolecule metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0008152" target="_blank">GO:0008152</a></td><td align="center">0.82</td><td>metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0080090" target="_blank">GO:0080090</a></td><td align="center">0.77</td><td>regulation of primary metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0031323" target="_blank">GO:0031323</a></td><td align="center">0.77</td><td>regulation of cellular metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0009987" target="_blank">GO:0009987</a></td><td align="center">0.52</td><td>cellular process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044260" target="_blank">GO:0044260</a></td><td align="center">0.36</td><td>cellular macromolecule metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044238" target="_blank">GO:0044238</a></td><td align="center">0.35</td><td>primary metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0048518" target="_blank">GO:0048518</a></td><td align="center">0.32</td><td>positive regulation of biological process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0048522" target="_blank">GO:0048522</a></td><td align="center">0.31</td><td>positive regulation of cellular process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0009893" target="_blank">GO:0009893</a></td><td align="center">0.29</td><td>positive regulation of metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0031325" target="_blank">GO:0031325</a></td><td align="center">0.28</td><td>positive regulation of cellular metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0010604" target="_blank">GO:0010604</a></td><td align="center">0.28</td><td>positive regulation of macromolecule metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0090304" target="_blank">GO:0090304</a></td><td align="center">0.18</td><td>nucleic acid metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051179" target="_blank">GO:0051179</a></td><td align="center">0.18</td><td>localization</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0032268" target="_blank">GO:0032268</a></td><td align="center">0.18</td><td>regulation of cellular protein metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051649" target="_blank">GO:0051649</a></td><td align="center">0.17</td><td>establishment of localization in cell</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051641" target="_blank">GO:0051641</a></td><td align="center">0.17</td><td>cellular localization</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051128" target="_blank">GO:0051128</a></td><td align="center">0.17</td><td>regulation of cellular component organization</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0006810" target="_blank">GO:0006810</a></td><td align="center">0.17</td><td>transport</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0071702" target="_blank">GO:0071702</a></td><td align="center">0.16</td><td>organic substance transport</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0046907" target="_blank">GO:0046907</a></td><td align="center">0.16</td><td>intracellular transport</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0032270" target="_blank">GO:0032270</a></td><td align="center">0.15</td><td>positive regulation of cellular protein metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0031399" target="_blank">GO:0031399</a></td><td align="center">0.15</td><td>regulation of protein modification process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0016070" target="_blank">GO:0016070</a></td><td align="center">0.15</td><td>RNA metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0006406" target="_blank">GO:0006406</a></td><td align="center">0.15</td><td>mRNA export from nucleus</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:1902275" target="_blank">GO:1902275</a></td><td align="center">0.14</td><td>regulation of chromatin organization</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051130" target="_blank">GO:0051130</a></td><td align="center">0.14</td><td>positive regulation of cellular component organization</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044699" target="_blank">GO:0044699</a></td><td align="center">0.14</td><td>single-organism process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0042127" target="_blank">GO:0042127</a></td><td align="center">0.14</td><td>regulation of cell proliferation</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0031401" target="_blank">GO:0031401</a></td><td align="center">0.14</td><td>positive regulation of protein modification process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0031062" target="_blank">GO:0031062</a></td><td align="center">0.13</td><td>positive regulation of histone methylation</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0010468" target="_blank">GO:0010468</a></td><td align="center">0.13</td><td>regulation of gene expression</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0008284" target="_blank">GO:0008284</a></td><td align="center">0.13</td><td>positive regulation of cell proliferation</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051171" target="_blank">GO:0051171</a></td><td align="center">0.09</td><td>regulation of nitrogen compound metabolic process</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_BP.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Biological Process. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.08,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>


<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_CC.svg" target="_blank">
    <img width=450 height=450 border="1" style="border-color:black" alt="GOsearchresult_final_CC.svg" src="GOsearchresult_final_CC.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Cellular Component (CC)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044428" target="_blank">GO:0044428</a></td><td align="center">1.00</td><td>nuclear part</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0032991" target="_blank">GO:0032991</a></td><td align="center">0.93</td><td>macromolecular complex</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043234" target="_blank">GO:0043234</a></td><td align="center">0.85</td><td>protein complex</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043229" target="_blank">GO:0043229</a></td><td align="center">0.59</td><td>intracellular organelle</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0005654" target="_blank">GO:0005654</a></td><td align="center">0.45</td><td>nucleoplasm</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043231" target="_blank">GO:0043231</a></td><td align="center">0.40</td><td>intracellular membrane-bounded organelle</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044451" target="_blank">GO:0044451</a></td><td align="center">0.15</td><td>nucleoplasm part</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_CC.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Cellular Component. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.14,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>

</td></tr>
</table>
</ul>




</font>


<hr>
<b>Reference:</b>
<ul><li>Chengxin Zhang, Peter L Freddolino, and Yang Zhang. (2018) <i>MetaGO: A Structure and Protein-Protein Interaction Network Based Pipeline for Automated Protein Function Annotations</i>. Journal of Molecular Biology.</ul>
[<a href="http://zhanglab.ccmb.med.umich.edu/MetaGO">back to server</a>]<br/>
</body></html>
