<html>
<head><meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>MetaGO result for PF11149</title></head>
<body>
<script type="text/javascript" src="/jmol/JSmol.min.js"></script>
<script type="text/javascript" src="/jmol/Jmol2.js"></script>
[<a href="http://zhanglab.ccmb.med.umich.edu/MetaGO">back to server</a>]<br/>
<h1 align="center">MetaGO result for job id PF11149</h1>


<font size="4" face="Arial">

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;User Input</font></b></div>
<ul>
    <table style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td colspan="2">&gt;UniRef100_A0A1M7SIR6/7-145 (139 residues)<br>
    IPARLAALKTMKTSELKAKWRELFDSEPPPFNRRYLESRLAYRIQELAYGGLKPETVRRL<br>
EKLGEDLDGGDRKKSRVRADLTPIAGTRLIREWQGVEHVVTVTADGFEWQGRPYKSLSAI<br>
ARAITGTRWNGWVFFGLKN<br>
    </td></tr>
    </table><br>

    <font size=2><br>
    Download query <a href="seq.fasta" download>sequence</a> 
    and <a href="input.pdb.gz" download>structure</a><br>
    </font>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Proteins with similar structure</font></b></div>
<ul>
    <table border="0" style="background:#F2F2F2";align="left">
        <tr><td align ="center"><script type="text/javascript">
            jmolInitialize("/jmol");
            jmolSetAppletColor("#000000");
            jmolApplet(450, "load model1_5cwpA.pdb.gz;set frank off;select all;spacefill off;wireframe off;select :T;trace 0.35;color trace mediumpurple;select :Q;cartoons;color group;set ambient 40;");
        </script><br/><script type="text/javascript">
            jmolCheckbox("set spin x 10;set spin y 10;spin on","spin off","Spin On/Off",false)
        </script></td>
        <td valign="top"><b>Top 10 structural analogs in PDB (as identified by
        <a href="http://zhanglab.ccmb.med.umich.edu/TM-align" target="_blank">
        TM-align</a>)</b><br/><br/>
        <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><th>Click<br>to view</th><th>Rank</th><th>PDB<br/>Hit</th><th>TM-score</th><th>RMSD<sup>a</sup></th><th>IDEN<sup>a</sup></th><th>Cov.</th><th>Download<br/>Alignment</th></tr>
        
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_5cwpA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">1</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5cwp" target="_blank"><font face="Monospace">5cwpA</font></a></td><td align="center">0.54</td><td valign="middle" align="center">3.65</td><td valign="middle" align="center">0.088</td><td valign="middle">0.755</td><td><a href="model1_5cwpA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_5cwiA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">2</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5cwi" target="_blank"><font face="Monospace">5cwiA</font></a></td><td align="center">0.51</td><td valign="middle" align="center">3.89</td><td valign="middle" align="center">0.066</td><td valign="middle">0.741</td><td><a href="model1_5cwiA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_3ibjA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">3</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3ibj" target="_blank"><font face="Monospace">3ibjA</font></a></td><td align="center">0.50</td><td valign="middle" align="center">4.44</td><td valign="middle" align="center">0.015</td><td valign="middle">0.777</td><td><a href="model1_3ibjA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_5cwoA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">4</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5cwo" target="_blank"><font face="Monospace">5cwoA</font></a></td><td align="center">0.50</td><td valign="middle" align="center">3.66</td><td valign="middle" align="center">0.067</td><td valign="middle">0.734</td><td><a href="model1_5cwoA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_3g45A.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">5</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3g45" target="_blank"><font face="Monospace">3g45A</font></a></td><td align="center">0.46</td><td valign="middle" align="center">4.88</td><td valign="middle" align="center">0.053</td><td valign="middle">0.784</td><td><a href="model1_3g45A.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_3ecmA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">6</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3ecm" target="_blank"><font face="Monospace">3ecmA</font></a></td><td align="center">0.46</td><td valign="middle" align="center">4.69</td><td valign="middle" align="center">0.076</td><td valign="middle">0.763</td><td><a href="model1_3ecmA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_2ouyA1.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">7</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2ouy" target="_blank"><font face="Monospace">2ouyA1</font></a></td><td align="center">0.43</td><td valign="middle" align="center">4.76</td><td valign="middle" align="center">0.009</td><td valign="middle">0.748</td><td><a href="model1_2ouyA1.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_3gdeA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">8</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3gde" target="_blank"><font face="Monospace">3gdeA</font></a></td><td align="center">0.41</td><td valign="middle" align="center">4.98</td><td valign="middle" align="center">0.082</td><td valign="middle">0.741</td><td><a href="model1_3gdeA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_3v5qA2.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">9</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3v5q" target="_blank"><font face="Monospace">3v5qA2</font></a></td><td align="center">0.37</td><td valign="middle" align="center">5.43</td><td valign="middle" align="center">0.057</td><td valign="middle">0.712</td><td><a href="model1_3v5qA2.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_1t9sA1.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">10</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1t9s" target="_blank"><font face="Monospace">1t9sA1</font></a></td><td align="center">0.36</td><td valign="middle" align="center">5.15</td><td valign="middle" align="center">0.053</td><td valign="middle">0.640</td><td><a href="model1_1t9sA1.pdb.gz" download>Download</a></td></tr>
        </table><br/>

        <table border="0" cellspacing="2" style="font-family:Arial;font-size:12px;background:#F2F2F2;">
            <tr><td valign="top">(a)</td><td align="justify">Query structure is shown in cartoon, while the structural analog is displayed using backbone trace.</td></tr>
            <tr><td valign="top">(b)</td><td align="justify">Ranking of proteins is based on TM-score of the structural alignment between the query structure and known structures in the PDB library.</td></tr>
            <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
            <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
            <tr><td valign="top">(e)</td><td align="justify">Cov. represents the coverage of the alignment by TM-align and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
        </table>
    </td></tr>
    </table>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Gene Ontology (GO) Terms</font></b></div>
<ul>
<table border="0" style="background:#F2F2F2";align="left"><tr><td>

<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_MF.svg" target="_blank">
    <img width=450 height=450 border="1" style="border-color:black" alt="GOsearchresult_final_MF.svg" src="GOsearchresult_final_MF.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Molecular Function (MF)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0004112" target="_blank">GO:0004112</a></td><td align="center">0.06</td><td>cyclic-nucleotide phosphodiesterase activity</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043169" target="_blank">GO:0043169</a></td><td align="center">0.04</td><td>cation binding</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_MF.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Molecular Function. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.13,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>


<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_BP.svg" target="_blank">
    <img width=450 height=450 border="1" style="border-color:black" alt="GOsearchresult_final_BP.svg" src="GOsearchresult_final_BP.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Biological Process (BP)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051716" target="_blank">GO:0051716</a></td><td align="center">0.04</td><td>cellular response to stimulus</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0050794" target="_blank">GO:0050794</a></td><td align="center">0.04</td><td>regulation of cellular process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044700" target="_blank">GO:0044700</a></td><td align="center">0.04</td><td>single organism signaling</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0007154" target="_blank">GO:0007154</a></td><td align="center">0.04</td><td>cell communication</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0046068" target="_blank">GO:0046068</a></td><td align="center">0.02</td><td>cGMP metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0009214" target="_blank">GO:0009214</a></td><td align="center">0.02</td><td>cyclic nucleotide catabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0009154" target="_blank">GO:0009154</a></td><td align="center">0.02</td><td>purine ribonucleotide catabolic process</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_BP.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Biological Process. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.08,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>


<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_CC.svg" target="_blank">
    <img width=450 height=450 border="1" style="border-color:black" alt="GOsearchresult_final_CC.svg" src="GOsearchresult_final_CC.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Cellular Component (CC)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0005829" target="_blank">GO:0005829</a></td><td align="center">0.02</td><td>cytosol</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0098590" target="_blank">GO:0098590</a></td><td align="center">0.01</td><td>plasma membrane region</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043204" target="_blank">GO:0043204</a></td><td align="center">0.01</td><td>perikaryon</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043025" target="_blank">GO:0043025</a></td><td align="center">0.01</td><td>neuronal cell body</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0005891" target="_blank">GO:0005891</a></td><td align="center">0.01</td><td>voltage-gated calcium channel complex</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0005813" target="_blank">GO:0005813</a></td><td align="center">0.01</td><td>centrosome</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_CC.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Cellular Component. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.14,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>

</td></tr>
</table>
</ul>




</font>


<hr>
<b>Reference:</b>
<ul><li>Chengxin Zhang, Peter L Freddolino, and Yang Zhang. (2018) <i>MetaGO: A Structure and Protein-Protein Interaction Network Based Pipeline for Automated Protein Function Annotations</i>. Journal of Molecular Biology.</ul>
[<a href="http://zhanglab.ccmb.med.umich.edu/MetaGO">back to server</a>]<br/>
</body></html>
