<html>
<head><meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>MetaGO result for PF12996</title></head>
<body>
<script type="text/javascript" src="/jmol/JSmol.min.js"></script>
<script type="text/javascript" src="/jmol/Jmol2.js"></script>
[<a href="http://zhanglab.ccmb.med.umich.edu/MetaGO">back to server</a>]<br/>
<h1 align="center">MetaGO result for job id PF12996</h1>


<font size="4" face="Arial">

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;User Input</font></b></div>
<ul>
    <table style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td colspan="2">&gt;UniRef100_L5JXA9/1-338 (338 residues)<br>
    MVNTGLAQASPLALANIGLAQVRALALASMGLDQVRALALDNTGLAQARPLALANMGLAQ<br>
VRALVLANMGPGQASPLALANTGLAQASPLALVNTGLAQDRALALANMDLDQASHLASVN<br>
MDLAQASPLALANTGLAQVRALALANMGLAQVRALPLATMGLDQASHLALVNMGLAQVRA<br>
LPLATMGLDQANHLALVNMGLAQVRALPLATMGLDQANHLALANMGLAQVRALPLATMGL<br>
DQANHLALVNMGLAQVRALPLATMGLDQANHLALVNMGLAQVRALPLATMGLDQANHLAL<br>
VNMGLAQVRALPLATMGLDQANHLALVNMGLAQVRALP<br>
    </td></tr>
    </table><br>

    <font size=2><br>
    Download query <a href="seq.fasta" download>sequence</a> 
    and <a href="input.pdb.gz" download>structure</a><br>
    </font>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Proteins with similar structure</font></b></div>
<ul>
    <table border="0" style="background:#F2F2F2";align="left">
        <tr><td align ="center"><script type="text/javascript">
            jmolInitialize("/jmol");
            jmolSetAppletColor("#000000");
            jmolApplet(450, "load model1_2qj0A.pdb.gz;set frank off;select all;spacefill off;wireframe off;select :T;trace 0.35;color trace mediumpurple;select :Q;cartoons;color group;set ambient 40;");
        </script><br/><script type="text/javascript">
            jmolCheckbox("set spin x 10;set spin y 10;spin on","spin off","Spin On/Off",false)
        </script></td>
        <td valign="top"><b>Top 10 structural analogs in PDB (as identified by
        <a href="http://zhanglab.ccmb.med.umich.edu/TM-align" target="_blank">
        TM-align</a>)</b><br/><br/>
        <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><th>Click<br>to view</th><th>Rank</th><th>PDB<br/>Hit</th><th>TM-score</th><th>RMSD<sup>a</sup></th><th>IDEN<sup>a</sup></th><th>Cov.</th><th>Download<br/>Alignment</th></tr>
        
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_2qj0A.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">1</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2qj0" target="_blank"><font face="Monospace">2qj0A</font></a></td><td align="center">0.39</td><td valign="middle" align="center">7.17</td><td valign="middle" align="center">0.053</td><td valign="middle">0.710</td><td><a href="model1_2qj0A.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_1t3tA2.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">2</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1t3t" target="_blank"><font face="Monospace">1t3tA2</font></a></td><td align="center">0.39</td><td valign="middle" align="center">6.77</td><td valign="middle" align="center">0.040</td><td valign="middle">0.675</td><td><a href="model1_1t3tA2.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_2uvfA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">3</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2uvf" target="_blank"><font face="Monospace">2uvfA</font></a></td><td align="center">0.38</td><td valign="middle" align="center">6.97</td><td valign="middle" align="center">0.040</td><td valign="middle">0.672</td><td><a href="model1_2uvfA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_3vstA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">4</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3vst" target="_blank"><font face="Monospace">3vstA</font></a></td><td align="center">0.38</td><td valign="middle" align="center">6.43</td><td valign="middle" align="center">0.034</td><td valign="middle">0.618</td><td><a href="model1_3vstA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_1czfA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">5</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1czf" target="_blank"><font face="Monospace">1czfA</font></a></td><td align="center">0.35</td><td valign="middle" align="center">5.94</td><td valign="middle" align="center">0.053</td><td valign="middle">0.530</td><td><a href="model1_1czfA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_1r9jB.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">6</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1r9j" target="_blank"><font face="Monospace">1r9jB</font></a></td><td align="center">0.32</td><td valign="middle" align="center">7.00</td><td valign="middle" align="center">0.085</td><td valign="middle">0.580</td><td><a href="model1_1r9jB.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_4fg5A.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">7</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4fg5" target="_blank"><font face="Monospace">4fg5A</font></a></td><td align="center">0.32</td><td valign="middle" align="center">6.64</td><td valign="middle" align="center">0.052</td><td valign="middle">0.547</td><td><a href="model1_4fg5A.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_1xnjB.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">8</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1xnj" target="_blank"><font face="Monospace">1xnjB</font></a></td><td align="center">0.31</td><td valign="middle" align="center">6.72</td><td valign="middle" align="center">0.055</td><td valign="middle">0.541</td><td><a href="model1_1xnjB.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_1rmgA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">9</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1rmg" target="_blank"><font face="Monospace">1rmgA</font></a></td><td align="center">0.31</td><td valign="middle" align="center">6.74</td><td valign="middle" align="center">0.046</td><td valign="middle">0.530</td><td><a href="model1_1rmgA.pdb.gz" download>Download</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_1bheA1.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">10</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1bhe" target="_blank"><font face="Monospace">1bheA1</font></a></td><td align="center">0.30</td><td valign="middle" align="center">6.30</td><td valign="middle" align="center">0.056</td><td valign="middle">0.503</td><td><a href="model1_1bheA1.pdb.gz" download>Download</a></td></tr>
        </table><br/>

        <table border="0" cellspacing="2" style="font-family:Arial;font-size:12px;background:#F2F2F2;">
            <tr><td valign="top">(a)</td><td align="justify">Query structure is shown in cartoon, while the structural analog is displayed using backbone trace.</td></tr>
            <tr><td valign="top">(b)</td><td align="justify">Ranking of proteins is based on TM-score of the structural alignment between the query structure and known structures in the PDB library.</td></tr>
            <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
            <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
            <tr><td valign="top">(e)</td><td align="justify">Cov. represents the coverage of the alignment by TM-align and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
        </table>
    </td></tr>
    </table>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Gene Ontology (GO) Terms</font></b></div>
<ul>
<table border="0" style="background:#F2F2F2";align="left"><tr><td>

<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_MF.svg" target="_blank">
    <img width=450 height=450 border="1" style="border-color:black" alt="GOsearchresult_final_MF.svg" src="GOsearchresult_final_MF.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Molecular Function (MF)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043169" target="_blank">GO:0043169</a></td><td align="center">0.03</td><td>cation binding</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0004872" target="_blank">GO:0004872</a></td><td align="center">0.02</td><td>receptor activity</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0019903" target="_blank">GO:0019903</a></td><td align="center">0.01</td><td>protein phosphatase binding</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0001875" target="_blank">GO:0001875</a></td><td align="center">0.01</td><td>lipopolysaccharide receptor activity</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0001530" target="_blank">GO:0001530</a></td><td align="center">0.01</td><td>lipopolysaccharide binding</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_MF.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Molecular Function. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.13,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>


<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_BP.svg" target="_blank">
    <img width=450 height=450 border="1" style="border-color:black" alt="GOsearchresult_final_BP.svg" src="GOsearchresult_final_BP.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Biological Process (BP)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044699" target="_blank">GO:0044699</a></td><td align="center">0.03</td><td>single-organism process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0009987" target="_blank">GO:0009987</a></td><td align="center">0.03</td><td>cellular process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0050896" target="_blank">GO:0050896</a></td><td align="center">0.02</td><td>response to stimulus</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0050794" target="_blank">GO:0050794</a></td><td align="center">0.02</td><td>regulation of cellular process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0048584" target="_blank">GO:0048584</a></td><td align="center">0.02</td><td>positive regulation of response to stimulus</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044763" target="_blank">GO:0044763</a></td><td align="center">0.02</td><td>single-organism cellular process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:2001013" target="_blank">GO:2001013</a></td><td align="center">0.01</td><td>epithelial cell proliferation involved in renal tubule morphogenesis</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:2000331" target="_blank">GO:2000331</a></td><td align="center">0.01</td><td>regulation of terminal button organization</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:1990268" target="_blank">GO:1990268</a></td><td align="center">0.01</td><td>response to gold nanoparticle</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:1901215" target="_blank">GO:1901215</a></td><td align="center">0.01</td><td>negative regulation of neuron death</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_BP.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Biological Process. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.08,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>


<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_CC.svg" target="_blank">
    <img width=450 height=450 border="1" style="border-color:black" alt="GOsearchresult_final_CC.svg" src="GOsearchresult_final_CC.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Cellular Component (CC)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0020036" target="_blank">GO:0020036</a></td><td align="center">0.01</td><td>Maurer's cleft</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0020003" target="_blank">GO:0020003</a></td><td align="center">0.01</td><td>symbiont-containing vacuole</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0005737" target="_blank">GO:0005737</a></td><td align="center">0.01</td><td>cytoplasm</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_CC.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Cellular Component. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.14,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>

</td></tr>
</table>
</ul>




</font>


<hr>
<b>Reference:</b>
<ul><li>Chengxin Zhang, Peter L Freddolino, and Yang Zhang. (2018) <i>MetaGO: A Structure and Protein-Protein Interaction Network Based Pipeline for Automated Protein Function Annotations</i>. Journal of Molecular Biology.</ul>
[<a href="http://zhanglab.ccmb.med.umich.edu/MetaGO">back to server</a>]<br/>
</body></html>
